src/Umbraco.Web/UmbracoHelper.cs

```diff
@@ -114,6 +114,14 @@ public IDataTypeService DataTypeService
         }
 
         /// <summary>
+        /// Lazy instantiates the IEntityService
+        /// </summary>
+        public IEntityService EntityService

```

Comment:
Only thing is that this shouldn't be public, we don't want to expose the full IEntityService via the UmbracoHelper, we just need it to expose the `GetIdForUdi` method on the UmbracoHelper.